conda/environments/cudf_dev_cuda11.0.yml

```diff
@@ -41,8 +41,8 @@ dependencies:
   - mypy=0.782
   - typing_extensions
   - pre_commit
-  - dask==2021.4.0
-  - distributed>=2.22.0,<=2021.4.0
+  - dask==2021.5.0
```

Comment:
Can these and all the other ones be `>=2021.4.0,<=2021.5.0`?

Reply:
Done